lib/Message/Passing/Filter/Encoder/Sereal.pm

```diff
@@ -0,0 +1,92 @@
+package Message::Passing::Filter::Encoder::Sereal;
+use Moo;
+use MooX::Types::MooseLike::Base qw( Bool HasMethods );
```

Comment:
Why do you need MooX::Types::MooseLike::Base here ? Seems like you never use these types ?